sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -68,15 +81,22 @@ public List<String> inputCollectionNames() {
 
   @Override
   public List<String> outputCollectionNames() {
-    return Lists.newArrayList("OUTPUT");
+    return Lists.newArrayList("output");
```

Comment:
Just wondering the purpose of UPPER to lower case change. Is it some cleanup?

Reply:
correct. Trying to standardize on lowercase